sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/ComputeManager.java

```diff
@@ -151,23 +163,35 @@ private ComputeManager(HttpPipeline httpPipeline, AzureProfile profile) {
         authorizationManager = AuthorizationManager.authenticate(httpPipeline, profile);
     }
 
-    /** @return the availability set resource management API entry point */
+    /**
+     * Gets the availability set resource management API entry point.
+     *
+     * @return the availability set resource management API entry point
+     */
     public AvailabilitySets availabilitySets() {
         if (availabilitySets == null) {
             availabilitySets = new AvailabilitySetsImpl(this);
         }
         return availabilitySets;
     }
 
-    /** @return the virtual machine resource management API entry point */
+    /**
+     * Gets the virtual machine resource management API entry point.
+     *
+     * @return the virtual machine resource management API entry point
+     */
     public VirtualMachines virtualMachines() {
         if (virtualMachines == null) {
             virtualMachines = new VirtualMachinesImpl(this, storageManager, networkManager, authorizationManager);
         }
         return virtualMachines;
     }
 
-    /** @return the virtual machine image resource management API entry point */
+    /**
+     * Gest the virtual machine image resource management API entry point.
```

Comment:
Gets

Reply:
Fixed by the [commit#68694ef](https://github.com/Azure/azure-sdk-for-java/pull/43499/commits/68694ef61ed373cc60dd24b9ea46f77420201faf)